doc/manuals/user/initial_notification.md

```diff
@@ -121,12 +121,15 @@ initial notification is required.
 Between both options, We have opted to implement the more flexible approach, which provide 
 the initial notification to users who can exploit is and can be ignore for who doesn't need it.
 
+For ignoring the initial notification, a URI parameter `options=skipInitialNotification can` be 
```

Comment:
I was thinking that the topic is important enough to deserver its own sub-topic in this file. Thus, I'd suggest the following wording (note it also include some fixing, you can just copy-paste):

```
## Avoid initial notification

For ignoring the initial notification, a URI parameter option `skipInitialNotification` can be used. Using this URI 
option, initial notification is always skipped at subscription creation/updation time. For example:

* `POST /v2/subscriptions?options=skipInitialNotification`
* `PATCH /v2/subscriptions/entities?options=skipInitialNotification`
```
```

Reply:
fixed in ab41005